doc/mediasdk-man.md

```diff
@@ -7898,6 +7902,68 @@ This structure is used to pass decryption status report index for Common Encrypt
 
 This structure is available since SDK API 1.30.
 
+## <a id='mfxExtAV1FilmGrainParam'>mfxExtAV1FilmGrainParam</a>
+
+**Definition**
+
+```C
+typedef struct {
+    mfxU8 Value;
+    mfxU8 Scaling;
+} mfxAV1FilmGrainPoint;
+
+typedef struct {
+    mfxExtBuffer Header;
+
+    mfxU16 Flags;
+    mfxU16 GrainSeed;
+
+    mfxU8  RefIdx;
+    mfxU8  NumYPoints;
+    mfxU8  NumCbPoints;
+    mfxU8  NumCrPoints;
+
+    mfxAV1FilmGrainPoint PointY[14];
+    mfxAV1FilmGrainPoint PointCb[10];
+    mfxAV1FilmGrainPoint PointCr[10];
+
+    mfxU8 GrainScalingMinus8;
+    mfxU8 ArCoeffLag;
+
+    mfxU8 ArCoeffsYPlus128[24];
+    mfxU8 ArCoeffsCbPlus128[25];
+    mfxU8 ArCoeffsCrPlus128[25];
+
+    mfxU8 ArCoeffShiftMinus6;
+    mfxU8 GrainScaleShift;
+
+    mfxU8  CbMult;
+    mfxU8  CbLumaMult;
+    mfxU16 CbOffset;
+
+    mfxU8  CrMult;
+    mfxU8  CrLumaMult;
+    mfxU16 CrOffset;
+
+    mfxU16 reserved[43];
+} mfxExtAV1FilmGrainParam;
+```
+
+**Description**
+
+This structure is used by AV1 SDK decoder to report film grain parameters for decoded frame. The application can attach this extended buffer to the [mfxFrameSurface1](#mfxFrameSurface) structure at runtime.
+
+**Members**
+
+| | |
+--- | ---
+`Header.BufferId` | Must be [MFX_EXTBUFF_AV1_FILM_GRAIN_PARAM](#ExtendedBufferID)
+`Value`, `Scaling`, `Flags`, `GrainSeed`, `RefIdx`, `NumYPoints`, `NumCbPoints`, `NumCrPoints`, `PointY`, `PointCb`, `PointCr`, `GrainScalingMinus8`, `ArCoeffLag`, `ArCoeffsYPlus128`, `ArCoeffsCbPlus128`, `ArCoeffsCrPlus128`, `ArCoeffShiftMinus6`, `GrainScaleShift`, `CbMult`, `CbLumaMult`, `CbOffset`, `CrMult`, `CrLumaMult`, `CrOffset` | These parameters represent film grain syntax of AV1 codec. They are equivalent of film_grain_params() from uncompressed header syntax of AV1 frame.<br><br>Parameter `Flags` is a bit map with bit-ORed flags from enum [AV1FilmGrainFlags](#AV1FilmGrainFlags)<br><br>If syntax parameter isn't present in uncompressed header of particular frame, respective member of `mfxExtAV1FilmGrainParam` is zeroed by the SDK AV1 decoder.
```

Comment:
Can you specify what each field means? The link to the spec might be used. The current explanation is too general

Reply:
Sure. I will add them based on AV1 spec.